server/apps/archived/archived.py

```diff
@@ -78,6 +92,116 @@ def on_create(self, docs):
                 if is_takes_package:
                     doc[SEQUENCE] = len(package_service.get_item_refs(doc))
 
+    def on_delete(self, doc):
```

Comment:
If the kill action is triggered via the master story then we should locate the broadcast stories and kill them as well.